charts/templates/_common.tpl

```diff
@@ -0,0 +1,25 @@
+{{- define "common.featureGate" -}}
```

Comment:
Usually apiserver and clustersynchro have different feature gates, so it is recommended to keep them separate